### ISSUE TITLE
Automatically assign a random reviewer to each new pull request

### ISSUE THREAD

**Issue by jdm** [servo/highfive#40]
Unless there's one selected specifically. We could base this off of the similar functionality in rust's highfive (https://github.com/nrc/highfive/blob/26d292ba3cd2a7ab38de0f0e7c8b6aff96c53a2a/highfive/newpr.py#L187), or build a simpler one. I'm not sure which is better.

**Comment by edunham** [comment]
From #servo: 
```
11:18:06 <~jdm> edunham: I don't think a refactor is necessary. It should just 
                be a new handler that assigns reviewers in order to address 
                https://github.com/servo/highfive/issues/40
11:18:39 <~jdm> edunham: oh, you mean have highfive respond to pull requests 
                for servo/homu?
11:18:57 < edunham> yep
11:19:03 <~jdm> ok, I understand now
11:19:11 <~jdm> yeah, there's an unfinished step in my highfive redesign
11:19:21 <~jdm> and that is moving the handlers out of highfive and into a 
                servo-highfive-handlers repo
11:20:29 <~jdm> and then you need to have a clone of highfive and a folder of 
                handlers (which might be a clone of another repo like 
                servo-highfive-handlers) for each group of repositories that 
                should share the same handlers
```

**Comment by rundrop1** [comment]
Claiming this issue.

**Comment by magni-** [comment]
Claiming this issue

**Comment by magni-** [comment]
This should have been fixed by #63 :)
